### DISCUSSION TITLE
Access related issue with User Name

### DISCUSSION THREAD

**Discussion by chilaka-priyanka** [community/community#68584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Team,

I have signed up into GitHub application with username , when Manager of the organsiation tries to provide me an access to the specific repository , he is not able to find the username.

Need your support on the same and get the issue resolved at the earliest.

Regards,
Priyanka

**Comment by vishalkrishna8** [comment]
Hello @chilaka-priyanka,
Here are steps you can take to help resolve the issue:
1.Double-check the spelling and capitalization of your username.
2.Confirm you're a member of the organization.
3.Ensure your manager has the right permissions.
4.Try using your email for the invitation.
5.Check organization settings for restrictions.
6.Verify the repository is in the correct organization.

Let me know if this was helpful to you.

**Reply by chilaka-priyanka** [reply]
Hi Vishal,
Is that possible to talk to you once?

**Reply by chilaka-priyanka** [reply]
How to validate the Point 2?

**Reply by vishalkrishna8** [reply]
If you are not a member, your manager may need to invite you to the organization first.

**Reply by chilaka-priyanka** [reply]
ok Vishal , let me check with my Manager once

**Reply by vishalkrishna8** [reply]
Sure, feel free to reach out if you need any additional information.

**Comment by chilaka-priyanka** [comment]
Hi

**Comment by chilaka-priyanka** [comment]
in the GitHub it is mentioned that presently i am not member of any organisation

**Comment by chilaka-priyanka** [comment]
can i tun that personal account to Organisation account with deloitte?

**Reply by vishalkrishna8** [reply]
If Deloitte doesn't already have an organization on GitHub, you can create one by going to your GitHub account settings and clicking on "Organizations" in the left sidebar. Then, click the "New organization" button and follow the steps to create it. You may need to use your company email address to verify your association with Deloitte.

**Reply by chilaka-priyanka** [reply]
i would like to have a call with you

**Reply by chilaka-priyanka** [reply]
is this possible?

**Reply by chilaka-priyanka** [reply]
i have followed this process and also i could see organisation is already avail.

**Comment by chilaka-priyanka** [comment]
> If Deloitte doesn't already have an organization on GitHub, you can create one by going to your GitHub account settings and clicking on "Organizations" in the left sidebar. Then, click the "New organization" button and follow the steps to create it. You may need to use your company email address to verify your association with Deloitte.

**Comment by cehfisher** [comment]
Hi @chilaka-priyanka - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
